.bumpversion.cfg

```diff
@@ -16,6 +16,8 @@ serialize =
 
 [bumpversion:file:octavia-cli/README.md]
 
+[bumpversion:file:run-ab-platform.sh]
```

Comment:
I have no idea why, but putting this at the bottom caused bumpversion to fail. It only worked when I put it here before the setup.py line